contrib/hived-config-patcher/src/patcher.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import sys
+import yaml
+import logging
+import argparse
+from kubernetes import client, config, watch
+
+
+class HiveDConfigPatcher(object):
+    def __init__(
+            self,
+            min_nodes=None,
```

Comment:
>min_nodes [](start = 12, length = 9)

Seems we do not use min_nodes

Reply:
it's an attribute of CA like max_nodes, included here in case we need it in the future